.gitignore

```diff
@@ -23,3 +23,4 @@ _testmain.go
 *.test
 *.prof
 *.bin
+.vscode
```

Comment:
LOL, if it bothers you that much, I am alright adding it but not sure it will cause many problems as is. :)